Mod2/Labs/Week1/VersionControl.md

```diff
@@ -0,0 +1,39 @@
+## Lab Objectives
+* Practice committing, branching, merging, and pushing.
+* Revisit some mod1 topics
+
+** Instructor Notes **  Before the lab, create groups of 2/3 people and post in slack.  Students will be working individually for the Practice portion with some group share-out, and then doing some individual work followed by some group work for the preparation section.
+
+## Practice
+
+### Git Flow Diagram
+Individually, you are going to diagram the git workflow that you might use when creating the program described below.  Be as specific as possible in your diagramming - write out the steps for each git process that you use.  Your diagram should include at least: 1 branch, 2 pushes, 5 commits.
+
+
+> ### E-Store
+> 
+> We are going to create the software system that will help a store manage it's inventory.  Your store can sell any kind of item you want.  
+> - You will need a class `Inventory` and at least two classes for the types of items the store sells.  For example, you could have `Inventory, Car, Motorcycle`, or `Inventory, Yarn, Knitting Needle`, or `Inventory, Book, Movie`.
+> - This project should be fully tested, with unit tests for each class.
+> - The methods and properties you create are up to you - if you need some inspiration or help coming up with methods, reach out to your instructor!
+
+After you have diagrammed your git workflow, you will discuss your workflow in small groups.  Pay close attention to how your processes differ from your cohort-mates.  There is no "right" way to complete this exercise, but there are some steps in the git workflow that are necessary; reach out to your instructors with **any** questions!
```

Comment:
I think this task will be pretty difficult. I think it will be a good challenge, but I'm in favor of a discussion here as a class so each student starts with a plan before implementing the workflow.